src/styles/components/SimpleTabs/_base.scss

```diff
@@ -56,8 +45,71 @@
   border-width: 1px;
   display: none;
   border-radius: get-border-radius(default);
+  border-top-left-radius: 0px;
 
   &#react-tabs-1 {
     border-top-left-radius: 0px;
   }
 }
+
+// colors
+.#{$ns}-simple-tab {
+  color: get-color(gray, 5);
+  background-color: get-color(gray, 1);
+
+  &.selected-tab {
+    color: get-color(gray, 6);
+    border-bottom-color: get-color(white);
+    background-color: get-color(white);
+
+    &:hover {
+      color: get-color(gray, 6);
+    }
+  }
+
+  &:hover {
+    color: get-color(black);
+  }
+
+  &.disabled-tab {
+    &:hover {
+      color: get-color(gray, 5);
+    }
+  }
+
+  &.#{$ns}-dark,
+  .#{$ns}-dark & {
+    color: get-color(gray, 5);
+    background-color: #293742;
+    border-color: rgba(16, 22, 26, 0.4);
+
+    &.selected-tab {
+      color: white;
+      background-color: #30404c;
```

Comment:
@casserni what do you want to do about these colors? https://github.com/stoplightio/ui-kit/pull/110/files#diff-5d757190a8e036540b65e00b1afcd45fR34

Reply:
Yeah since these colors are defined as blueprint variables and not taken from the theme we need to think of something either extend the theme to have more colors or rethink our strategy

https://github.com/stoplightio/ui-kit/blob/master/src/styles/blueprint/_base.scss#L34

Reply:
Moved these colors to variables